src/main/java/io/github/thebusybiscuit/slimefun4/utils/SlimefunUtils.java

```diff
@@ -486,7 +486,7 @@ private static boolean equalsItemMeta(@Nonnull ItemMeta itemMeta, @Nonnull ItemM
         }
 
         if (itemMeta instanceof PotionMeta && sfitemMeta instanceof PotionMeta) {
-            return ((PotionMeta) itemMeta).getBasePotionData().equals(((PotionMeta) sfitemMeta).getBasePotionData());
+            return ((PotionMeta) itemMeta).getBasePotionType().equals(((PotionMeta) sfitemMeta).getBasePotionType());
```

Comment:
`getBasePotionType` is added in 1.20.2, this code will break backward compatibility.
also, `getBasePotionType` becomes nullable in 1.20.5, need to check `hasBasePotionType` before.

This will break all potion items.

Reply:
There were multiple places where the new `PotionMeta#getBasePotionType` were used. I've refactored all of them to do a version check to ensure backwards compat.